acceptance/tests/resource/service/ticket_3927_redhat_systemd_vs_service_provider.rb

```diff
@@ -0,0 +1,34 @@
+test_name 'RedHat Service vs. Systemd Provider Validation'
+
+# A simple acceptance test to ensure basic usage of the service
+# provider works for a mix of sysV and systemd RedHat Linux platforms
+confine :to, :platform => /el\-5|el\-6|el\-7|centos\-5|centos\-6|centos\-7|fedora\-19|fedora\-20/
+
+manifest_install_httpd = %Q{
+  package { 'httpd':
+    ensure => present,
```

Comment:
@joshcooper yes, I made sure that the httpd service is available in the default repos for all the redhat platforms we're testing here.